### DISCUSSION TITLE
风月同天 | /cn/2020/07/same-world/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1429] [Comments]
https://yihui.org/cn/2020/07/same-world/

[a comment by an automated account is not shown]

**Reply by yihui** [reply]
画意可能是，诗情应该不一定。不是说国家不幸诗家幸么。还有欧阳修的《梅圣俞诗集序》说的也有一定的代表性：<https://yihui.org/cn/2018/07/fluent-essay/> 它的开头甚至提到世人的印象是“诗人少达而多穷”；放在当代，余秀华可能就是这样的例子：<https://www.liechi.org/cn/2019/12/poems_by_yu/>

当然，写诗是需要时间的，所以“不用忙忙碌碌”可能确实是个必要条件（也不排除有些诗人很厉害，一挥而就）。至于吃喝，我估计多数人应该不会真的缺了吧。

欣赏诗和写诗应该也差不多，确实不能太忙碌，不过也许有人会发现有些忙碌的事情不如欣赏诗，人太忙了会昏头，欣赏诗有助于镇定。

> ——原帖发布于 2021-08-10 14:02:57

[a comment by an automated account is not shown]

**Comment by pangpang-newer** [comment]
爲什麽你中文版跟英文版的内容不一樣？ 想問下什麽原因讓你這樣做？

**Reply by yihui** [reply]
因為中文是母語，可以讓我表達更多更細微的感性，而英文是工作語言，所以更多寫技術方面的內容。說白了，就是我英文能力有限，寫日常生活或所思所想會有點吃力。
